darts/models/standard_regression_model.py

```diff
@@ -18,65 +16,31 @@
 class StandardRegressionModel(RegressionModel):
```

Comment:
Could we rename `LinearRegressionModel` maybe?

Reply:
Yep I thought about it and that's probably better. Just wasn't sure to drop support for a class called `StandardRegressionModel` completely.
I just the file and class name.

Reply:
No I have 20 more changed file but that's due to the renaming from StandardRegressionModel to LinearRegressionModel.

Reply:
OK, I think it's OK to change the name :)